docs/guide/features.md

```diff
@@ -295,6 +295,7 @@ Note that:
 - This is a Vite-only feature and is not a web or ES standard.
 - The glob patterns are treated like import specifiers: they must be either relative (start with `./`) or absolute (start with `/`, resolved relative to project root).
 - The glob matching is done via `fast-glob` - check out its documentation for [supported glob patterns](https://github.com/mrmlnc/fast-glob#pattern-syntax).
+  - You should also be aware that glob imports do not accept variables, you need to directly pass the string pattern.
```

Comment:
```suggestion
- You should also be aware that glob imports do not accept variables, you need to directly pass the string pattern.
```

Reply:
Perhaps in that bullet possible solutions should be mentioned?

> you need to directly pass the string pattern, but this can be abstracted using various methods such as switch statements or objects

Reply:
The problem is that we need to describe then what is the "this" here. IMO, once you know that you can't use a variable, the user is going to be able to refactor their code as you properly did in the PR. Maybe this is good content for a tutorial, or a blog post? We need more of those in the community :)
Let's merge it so we get the note in that is going to surely help some users. Thanks for raising the issue and for the PR.